unittests/finalizer_tests.cpp

```diff
@@ -154,8 +168,308 @@ BOOST_AUTO_TEST_CASE( finalizer_safety_file_io ) try {
 
 } FC_LOG_AND_RETHROW()
 
```

Comment:
Do you want to add some tests for corrupted safety file?

Reply:
Yes, good suggestion, done.